ParallelPrimitives/RadixSort.cpp

```diff
@@ -5,13 +5,17 @@
 #include <numeric>
 
 #if defined( ORO_PP_LOAD_FROM_STRING )
-#include <ParallelPrimitives/cache/Kernels.h>
 #include <ParallelPrimitives/cache/KernelArgs.h>
+#include <ParallelPrimitives/cache/Kernels.h>
```

Comment:
Did you use the automatic include sorting? If I remember correctly, I had some issue with baking when I used the automatic include sorting. I think it must be in this order:
#include <ParallelPrimitives/cache/Kernels.h>
#include <ParallelPrimitives/cache/KernelArgs.h>

Reply:
I'll add 
```
// clang-format off
...
// clang-format on
```

to turn off formatting here